### PR TITLE
Travis build

### PR DESCRIPTION
**Pull request by manthey**
Basic server tests on travis.

### CODE REVIEW

**Comment by jeffbaumes** [comment]
Are errors like this supposed to be happening?

https://travis-ci.org/osumo/osumo/builds/144964779#L5093-L5119
https://travis-ci.org/osumo/osumo/builds/144964779#L5149-L5175
https://travis-ci.org/osumo/osumo/builds/144964779#L5206-L5233

**Comment by manthey** [comment]
The errors in the worker log are intentional -- it is testing that a job with bad parameters properly report errors.  I am printing the worker log, but that isn't isn't really necessary.

We are getting some SSL errors when cmake configures girder packaging tests (which we aren't running) , and I think those should be cleaned up.  I've worked around them for the moment by using `make || true`, but this is not the right way.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,120 @@
+# Use the R language default, as travis always has Python
+language: python
```
Comment (opadron) [inline .travis.yml:4]: Is this supposed to be set to "python" if we're running as an R project?

File: Gruntfile.js [hunk 2]
```diff
@@ -1,84 +1,82 @@
...
+  var path = require('path');
+  // var fs = require('fs');
 
     // This gruntfile is only designed to be used with girder's build system.
     // Fail if grunt is executed here.
-    if (path.resolve(__dirname) === path.resolve(process.cwd())) {
-        grunt.fail.fatal(
+  if (path.resolve(__dirname) === path.resolve(process.cwd())) {
+    grunt.fail.fatal(
             "To build Osumo, run grunt from Girder's root directory");
```
Comment (opadron) [inline Gruntfile.js]: (minor nit) this line should be indented like it were code.

File: Gruntfile.js [hunk 3]
```diff
@@ -1,84 +1,82 @@
...
-    grunt.config.merge({
-        shell: {
-            "osumo-npm": {
+  grunt.config.merge({
+    shell: {
+      'osumo-npm': {
                 /*
                  * TODO(opadron): remove the true part once webpack 2 finally
                  *                comes out of beta
                  */
```
Comment (opadron) [inline Gruntfile.js]: (minor nit) these comments should be indented like it were code.

File: .travis.yml (hunk 1 above)
Comment (manthey) [inline .travis.yml:4]: I initially tried R.  Fundamentally, we use R, python, javascript, and bash.  If you use the travis R environment, the python is harder to get to what we want then it was to specify python and get R where we want.  Python is a first-class citizen in travis, whereas R is still a second-class citizen.

Comment (manthey) [inline .travis.yml:4]: But I should have changed the comment after switching from R to python.  I'll do so.

**Comment by opadron** [comment]
LGTM

**Pull request merged**